DIMS/GenerateExcel.R

```diff
@@ -142,7 +168,7 @@ if (z_score == 1) {
   outlist[, intensity_col_ids][outlist[, intensity_col_ids] == 0] <- NA
 
   # save outlist as it is and use it to calculate robust scaler
-  outlist_noZ <- outlist
+  outlist_robustZ <- outlist_nooutliers <- outlist
```

Comment:
I have never seen a chained assignment like this before. It is interesting, but I think it would improve readability to split them up
```suggestion
  outlist_robustZ <- outlist
  outlist_nooutliers <- outlist
```